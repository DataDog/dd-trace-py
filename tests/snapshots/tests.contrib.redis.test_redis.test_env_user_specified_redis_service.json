[[
  {
    "name": "redis.command",
    "service": "myredis",
    "resource": "GET cheese",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "redis",
    "meta": {
      "_dd.base_service": "",
      "_dd.p.dm": "-0",
      "component": "redis",
      "db.system": "redis",
      "language": "python",
      "out.host": "localhost",
      "redis.raw_command": "GET cheese",
      "runtime-id": "e263ff9ad1cd43099216a11ca5e19377",
      "span.kind": "client"
    },
    "metrics": {
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "db.row_count": 0,
      "network.destination.port": 6379,
      "out.redis_db": 0,
      "process_id": 20046,
      "redis.args_length": 2
    },
    "duration": 501125,
    "start": 1692651821692035875
  }],
[
  {
    "name": "redis.command",
    "service": "cfg-redis",
    "resource": "GET cheese",
    "trace_id": 1,
    "span_id": 1,
    "parent_id": 0,
    "type": "redis",
    "meta": {
      "_dd.base_service": "",
      "_dd.p.dm": "-0",
      "component": "redis",
      "db.system": "redis",
      "language": "python",
      "out.host": "localhost",
      "redis.raw_command": "GET cheese",
      "runtime-id": "e263ff9ad1cd43099216a11ca5e19377",
      "span.kind": "client"
    },
    "metrics": {
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "db.row_count": 0,
      "network.destination.port": 6379,
      "out.redis_db": 0,
      "process_id": 20046,
      "redis.args_length": 2
    },
    "duration": 329333,
    "start": 1692651821722196292
  }]]
