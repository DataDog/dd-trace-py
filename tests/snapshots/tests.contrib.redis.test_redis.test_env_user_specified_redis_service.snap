[[{"name" "redis.command"
   "service" "myredis"
   "resource" "GET cheese"
   "type" "redis"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1596062188510557000
   "duration" 3028000
   "meta" {"runtime-id" "261677c51ca04356a3ef7bbd65295b8b"
           "redis.raw_command" "GET cheese"
           "out.host" "localhost"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 76423
              "out.port" 6379
              "out.redis_db" 0
              "redis.args_length" 2}}]
 [{"name" "redis.command"
   "service" "cfg-redis"
   "resource" "GET cheese"
   "type" "redis"
   "error" 0
   "span_id" 0
   "trace_id" 1
   "parent_id" nil
   "start" 1596062188513861000
   "duration" 979000
   "meta" {"runtime-id" "261677c51ca04356a3ef7bbd65295b8b"
           "redis.raw_command" "GET cheese"
           "out.host" "localhost"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 76423
              "out.port" 6379
              "out.redis_db" 0
              "redis.args_length" 2}}]]
